test/tests/functional/pbs_sister_mom_crashes.py

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2018 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+class TestSisterMom(TestFunctional):
+    """
+    This test suite tests the sister mom crash
+    """
+    @timeout(240)
+    def test_sister_mom_crash(self):
+        """
+        This test suite verifies that sister mom
+        doesn't crash
+        """
+        # Skip test if number of mom provided is not equal to two
+        if len(self.moms) != 2:
+            self.skipTest("test requires atleast two MoMs as input, " +
+                          "use -p moms=<mom1:mom2>")
+        sister_mom = self.moms.keys()[1]
+        self.script = "pbsdsh dd if=/dev/zero of=/dev/null"
+        j = Job(TEST_USER)
+        j.set_attributes({'Resource_List.select': '2',
+                          'Resource_List.place': 'scatter'})
+        j.create_script(self.script)
+        self.jid = self.server.submit(j)
+        self.server.expect(JOB, {'job_state': 'R'}, id=self.jid)
+        self.server.log_match(";%s;node down: communication closed"
+                              % (sister_mom),
+                              max_attempts=18, interval=10, existence=False)
+
+    def tearDown(self):
+        self.server.delete(self.jid)
```

Comment:
This isn't needed.  Make the test inclusive.  The job is short, it will end shortly after the test finishes.  If you try and run a new test, it will be deleted in setUp().